partcad/src/partcad/runtime.py

```
@@ -28,3 +59,156 @@ def __init__(self, ctx, name):
+                    pc_logging.debug("Container is starting...")
+                    time.sleep(1)
+
+            pc_logging.debug("Container properties are: %s" % container.attrs)
+            host = container.attrs["NetworkSettings"]["Networks"]["bridge"]["IPAddress"]
+            pc_logging.debug("The docker container is running at: %s" % host)
+        else:
+            raise Exception("Remote docker sandboxes are not supported yet")
+
+        self.rpc_client = RuntimeJsonRpcClient(host, port)
+
+    def run(self, cmd, stdin=None, cwd=None, input_files=[], output_files=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)